asserts/database.go

```diff
@@ -365,6 +386,11 @@ func (db *Database) Add(assert Assertion) error {
 		return fmt.Errorf("cannot add %q assertion with primary key clashing with a trusted assertion: %v", ref.Type.Name, ref.PrimaryKey)
 	}
 
+	_, err = db.predefined.Get(ref.Type, ref.PrimaryKey, ref.Type.MaxSupportedFormat())
+	if err != ErrNotFound {
+		return fmt.Errorf("cannot add %q assertion with primary key clashing with a predefined assertion: %v", ref.Type.Name, ref.PrimaryKey)
```

Comment:
I'd add a comma at the right spot:

`cannot add %q, assertion with primary key clashing with a predefined assertion: %v`